ourjapanlife/findadoc-frontend
# [READY] Autogenerate locales file before build (#80). This removes the hardco…

…ded lists in Toolbar and nuxt.config.js files

This change depends on the `file` key being added to the localization repo, [per this PR](https://github.com/ourjapanlife/findadoc-localization/pull/39/files).

## Resolves #80 

### Changes

Dynamically load language list instead of static lists in config files.

This PR has been updated to use the file generated in the localization repo as a source, instead of generation in this package.

### How to test

1. Pull, then `yarn build; yarn run` or `yarn dev`
2. Browse around and switch languages in the app - this should work exactly as before with no changes in functionality.

### Screenshots

<img width="412" alt="スクリーンショット 2021-06-20 15 32 19" src="https://user-images.githubusercontent.com/6155643/122664430-47a5f180-d1dc-11eb-85b8-abb5b9884ba8.png">
